src/runtime_src/core/tools/xbflash2/XBFUtilities.cpp

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
I thought the reason we separated xbutilties and xbutilitiescore was so that we won't need another file. Can you please explain why we have these 2 functions in this file?

Reply:
I don't see any xrt_core dependency in `XBUtilities::can_proceed` and the only dependency I see in `XBUtilities::sudo_or_throw` is error handling. Can't we change that to std::error. I don't think that adding this file is justified. 

Maybe @stsoe can chime in. 
Soren, what is the difference between:
`throw xrt_core::error(std::errc::operation_canceled);`
and 
`throw std::errc::operation_canceled;`

Reply:
Removed XBFUtilities files and using functions from tools/common framework.